app/views/claims/verified.html.erb

```diff
@@ -5,9 +5,28 @@
     </h1>
 
     <p class="govuk-body">
-      You can now continue with your claim
+      This is your <%= current_claim.presented_identity.verified_fields %> from your digital identity:
     </p>
 
+    <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+      <% [:full_name, :date_of_birth, :payroll_gender, :address].each do |k| %>
+        <% if current_claim.presented_identity.send(k) %>
+          <%= render partial: "answer_row", locals: { label: I18n.t("verified_fields.#{k}").capitalize, answer: current_claim.presented_identity.send(k), slug: nil } %>
+        <% end %>
+      <% end %>
```

Comment:
Was there a particular reason to do this in a different way from how we do the check your answers page? We already have all of the data and logic needed there.